lib/block_view.go

```diff
@@ -1669,7 +1669,11 @@ func (bav *UtxoView) DisconnectBlock(
 				if len(utxoOp.PrevStakeEntries) != 1 {
 					return fmt.Errorf("DisconnectBlock: Expected exactly one prev stake entry for stake distribution op")
 				}
+				if utxoOp.PrevValidatorEntry == nil {
+					return fmt.Errorf("DisconnectBlock: Expected prev validator entry for stake distribution op")
```

Comment:
Is it possible to hit this error case at all? Or is this more-so to bullet-proof the disconnect?

Reply:
Just to bullet proof disconnects.